src/StatusQ/Controls/StatusInput.qml

```diff
@@ -37,6 +37,7 @@ Item {
     property list<StatusAsyncValidator> asyncValidators
     property int validationMode: StatusInput.ValidationMode.OnlyWhenDirty
     property string validatedValue
+    property bool ignoreInvalidInput: false
```

Comment:
This is a cool feature. Could consider introducing a new `ValidationMode` for that instead of introducing a new prop, but I'm happy to keep this.